tutorretirement/patches/k8s-jobs

```diff
@@ -18,4 +18,12 @@ spec:
                 - 'bash'
                 - 'run_retirement_pipeline.sh'
                 - '{{ RETIREMENT_COOL_OFF_DAYS }}'
+              volumeMounts:
+                - name: pieline-config
```

Comment:
Typo. Though I really like the idea of a pieline, make mine blueberry please! :)

Reply:
haha it sound like a good line to stand in :yum: 
Great catch! I'll fix :wrench: